### ISSUE TITLE
Convention for views naming and @Template usage

### ISSUE THREAD

**Issue by BFoucher** [Victoire/victoire#1047]
## Description
When i upgrade my project from victoire 2.2 and symfony 2.8 to victoire 2.3 and symfony 3.4, i met some errors with template naming. For example in `BusinessTemplateController::listEntitiesAction` we use `@Template`annotation without specific path. The file is in `BusinessPageBundle\Resources\views\BusinessTemplate\listEntities.html.twig` and when i try to access of this method : 
```
Unable to find template "@VictoireBusinessPage/business_template/list_entities.html.twig" (looked into: ~/ProjectDir/vendor/victoire/victoire/Bundle/BusinessPageBundle/Resources/views).
```
Today Symfony best practice are to name template with snake_case. (http://symfony.com/doc/current/best_practices/templates.html#template-locations) 

## Resolution Ideas
We could rename all templates in snake_case and/or forbid usage of `@Template` annotation and be more explicit (http://symfony.com/doc/3.4/best_practices/controllers.html#template-configuration)

## Related to

| Bundle         |    --    | --             | --       |
| -------------- |:--------:| --             | --       |
| Analytics      |  | Page           | &#10004; |
| Blog           |          | Query          |          |
| BusinessEntity |          |Seo             |          |
| BusinessPage   |  &#10004;        | Sitemap        |          |
| Core           |    &#10004;      | Template       |   &#10004;       |
| Criteria       |          | Twig           |          |
| Filter         |          | User           |          |
| Form           |          | ViewReference  |          |
| I18n           |          | Widget         |          |
| Media          |          | WidgetMap      |          |
| UI             |          |                |          |

**Comment by alexislefebvre** [comment]
This should have been changed in #1044.

Maybe this file is not called during tests, so it threw no error and I didn't have to rename it.

You can use Victoire 2.3.31 as a temporary solution.

**Comment by BFoucher** [comment]
Thanks for 2.3.31 for quick fix, so for the moment victoire 2.3.32 is not full functional.
I'm think we need to find all @Template() usage and give the full path to prevent this problem.

We should respect best practices and just forbid this Annotation. We can add this in static test for identify all usage in new PR

**Comment by alexislefebvre** [comment]
In the same time, we'll have to explain in https://github.com/Victoire/victoire/blob/f55326f4a95815a74bfa0e0dc2a4798e87a81611/CHANGELOG.md#2332 that it only concern templates that are called by `@Template` (without parameter).
